pars-fmt/src/error.rs

```diff
@@ -27,6 +27,12 @@ impl FormatError {
     }
 }
 
+impl fmt::Debug for FormatError {
```

Comment:
I sort of think we should continue deriving `Debug` -- it would be more useful for debugging, and we should be printing with `Display` to users.

Reply:
The rationale for this is that when the compiler panics, it uses the `Debug` impl to print the message, and that's where we want our fancy formatting. The answer might be to make `Display` just be less fancy tho, will do that